newrelic/config.py

```diff
@@ -1850,15 +1864,19 @@ def _process_error_trace_configuration():
             (module, object_path) = function.split(":", 1)
 
             ignore_errors = []
+            expected_classes = []
 
             if _config_object.has_option(section, "ignore_errors"):
                 ignore_errors = _config_object.get(section, "ignore_errors").split()
```

Comment:
I assume then that we're renaming this setting in a future PR with all the associated rip out?

Reply:
Correct. This will get renamed in the work associated with deprecating the `ignore_errors` setting.